iotqatools/simulators/listen/listen.js

```diff
@@ -58,26 +99,37 @@ process.argv.forEach(function (val, index, array) {
                 console.log("# Accumulator server version: " + version );
                 process.exit(0);
         }
-        if (val=="-u"){
+        if (val=="-u" || val=="--help" ){
```

Comment:
These simulators have their own implicit documentation but could be useful to have all the info together. 
Task added #32